ssc/Pos/Arbitrary/Ssc.hs

```diff
@@ -177,19 +176,19 @@ instance HasConfiguration => Arbitrary VssCertificatesMap where
     shrink = genericShrink
 
 instance HasConfiguration => Arbitrary VssCertData where
-    arbitrary = makeSmall genericArbitrary
+    arbitrary = genericArbitrary
     shrink = genericShrink
 
 instance HasConfiguration => Arbitrary SscGlobalState where
-    arbitrary = makeSmall genericArbitrary
+    arbitrary = genericArbitrary
     shrink = genericShrink
 
 instance HasConfiguration => Arbitrary SscSecretStorage where
     arbitrary = genericArbitrary
     shrink = genericShrink
 
 instance HasConfiguration => Arbitrary TossModifier where
-    arbitrary = makeSmall genericArbitrary
+    arbitrary = genericArbitrary
```

Comment:
Did you measure how it affected tests running time? I mean all removals of `makeSmall`. I'm fine with these changes as long as tests don't become significantly slower.

Reply:
My solution was to attenuate the size at the test site using `modifyMaxSize` rather than build it into the arbitrary instance.

Reply:
The safecopy test would make my HDD light solid blue and I'd have to trigger the OOM killer to stop the test. Setting max size to 8 fixed that.